fiasco/conftest.py

```diff
@@ -48,8 +48,15 @@
     'fe_18.rrparams':                       '6a36e5db1e377d0c30652960eb77b93d',
     'fe_18.drparams':                       '1e8b0194aeb099b2653508110e388200',
     'fe_27.rrparams':                       '75383b0f1b167f862cfd26bbadd2a029',
-    'fe_10.psplups':                        'dd34363f6daa81dbf106fbeb211b457d',
```

Comment:
This file is used in the `test_proton_collision` test.

Reply:
If you want, just replace the `fe10` ion fixture with `mg5` and then use that for the proton collision rate and the new level population test. There's no reason to have both.